src/pages/guides/FAQ/FAQ -11-24

```diff
@@ -0,0 +1,58 @@
+* What are **Collections** in V4?
+   
+   Collections are flexible, real-time, saved views of a selection of assets that allow users to dynamically select, filter, group, and sort their media using previously tagged metadata. Collections allow users to build meaningful, efficient workflows without the tool forcing them to work within a particular approach, process or template.
+* Is the Frame V4 API **backward compatible**?  
+    No. The Frame product released at Adobe MAX operates in fundamentally different ways and emphasizes different workflows than the legacy version. Due to these differences, the new API is not backward compatible with the Legacy Product.
+
+        The Frame.io V4 API is *not* backward-compatible with earlier versions of Frame.io APIs and, generally, cannot be used to access or update resources contained within legacy accounts as there have been significant changes to the V4 concepts and data model. As such, the URIs associated with the V4 API all include a `/v4` path prefix. However, the V4 API is still evolving quickly and it is possible that new features may occasionally warrant breaking changes.  
+
+    Recognizing that backward-compatibility is a major concern for customers managing production-quality integrations with high uptime requirements, we are designing the V4 API to support an additional level of versioning via a custom HTTP header in order to allow clients to opt into using experimental endpoints, avoid breaking changes, and provide backward-compatibility guarantees within the V4 namespace. 
+
+* Will the V4 API only be **available for enterprise**? 
+   
+   For the moment, Yes. Once the V4 API is generally available, will the automations I set up on V3 using Developer Tokens be broken, or will these be migrated to V4 automatically? Will I need to start my automations from scratch?
+*2.* Once the V4 API is generally available, will the **automations I set up on V3** using developer tokens be broken, or will these be migrated to V4 automatically? 
+   All automations using v3 developer tokens will be broken and we **do not** have any way to migrate them to v4.
+* Can we create and **manage Collections programmatically** on V4?
+    1. We do not yet have any Collections endpoints publicly available. But, Collections management is under development.
+* Will there be a **Zapier** connection available for V4? 
+    1. The existing Zapier integration however **does not work** with V4 yet. It likely will in the future, but I don’t think commitments have actually been made by anyone.
+* Is there support or recommended workflows for integrating [Frame.io](http://frame.io/) V4 with Airtable, [Descript.com](http://descript.com/), and [Kitchn.io](http://kitchn.io/)?
+     Not yet.
```

Comment:
I think we should say it's in development if we expect that we'll be updating these integrations in the relatively near future. Otherwise we should be clear that this is something that we expect the community to contribute.